test/e2e/framework.go

```diff
@@ -1527,6 +1527,21 @@ func (data *TestData) createNginxPodOnNode(name string, ns string, nodeName stri
 	}).WithHostNetwork(hostNetwork).Create(data)
 }
 
+func (data *TestData) createUDPServerPod(name string, ns string, portNum int32, serverNode string) error {
+	cmd := []string{"/bin/bash", "-c"}
```

Comment:
you don't need to create a shell to run `/agnhost`
And the default command is already `/agnhost` so you just need to specify args: `WithArgs([]string{"serve-hostname", "--udp", "--http=false", "--port", fmt.Sprint(portNum)})`

Reply:
updated.